Plugin.php

```diff
@@ -105,6 +105,30 @@ public function boot()
             EventRegistry::instance()->setMessageContext($page);
         }, 100);
 
+        /*
+         * Populate MenuItem properties with localized values if available
+         */
+        Event::listen('pages.menu.referencesGenerated', function (&$items) {
+            $iterator = function ($menuItems) use (&$iterator) {
+                $result = [];
+                $locale = App::getLocale();
```

Comment:
Why is this in an iterator instead of just a foreach loop in the event handler?

Reply:
This is to handle the subitems, see below:
```
                    if ($item->items) {
                        $item->items = $iterator($item->items);
                    }
```

Reply:
ref. https://github.com/rainlab/pages-plugin/blob/master/classes/Menu.php#L277-L292

Reply:
Gotcha. Can we make the `App::getLocale()` call outside of the iterator then?